javascript/modules/client/src/client.ts

```diff
@@ -253,7 +255,7 @@ export class Client extends ClientCore implements IClient {
      * @return {*}  {IPermissionParams}
      * @memberof Client
      */
-    grantAction: (data: Uint8Array): IPermissionParams =>
+    grantAction: (data: Uint8Array): IPermissionResponse =>
```

Comment:
Response makes sense if there was a request, I'm not sure `{ who, where, permission, ... }` is a response to something

Reply:
How about `...DecodedParams`?

Reply:
Seems fine to me :+1: